### PR TITLE
Priority for already loaded record

### PR DESCRIPTION
**Pull request by lifeart**
if we already have loaded recored - do not reload it from server

### CODE REVIEW

File: app/services/filter-data.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Service from '@ember/service';
 
 export default Service.extend({
-  showInherited: false,
+  showInherited: true,
```
Comment (sivakumar-kailasam) [inline app/services/filter-data.js]: we wouldn't want to change this behaviour for now

Comment (lifeart) [reply app/services/filter-data.js]: @sivakumar-kailasam if I set it to `false`, 4 tests fail and this is magic for me..

Comment (lifeart) [reply app/services/filter-data.js]: @sivakumar-kailasam done

**Comment by lifeart** [comment]
@Turbo87

File: app/routes/project-version/classes/class.js [hunk 2]
```diff
@@ -64,7 +69,7 @@ export default Route.extend(ScrollTracker, {
 
   serialize(model) {
     return {
-      class: model.get('name')
+      class: get(model, 'name')
```
Comment (Turbo87) [inline app/routes/project-version/classes/class.js]: Why did you change this?

Comment (lifeart) [reply app/routes/project-version/classes/class.js]: @Turbo87 `class` route alteady has `import { set, get } from '@ember/object';`, why not to use them?

Comment (Turbo87) [reply app/routes/project-version/classes/class.js]: because they are much harder to read IMHO and only make sense if `model` was a POJO and not an `Ember.Object`

Comment (lifeart) [reply app/routes/project-version/classes/class.js]: Rolled-back

**Comment by toddjordan** [comment]
I might be off base here, but doesn't ember-data implicitly do caching?  findRecord should load existing items in the store if already loaded.

**Comment by lifeart** [comment]
@toddjordan looks like it's depends on adapter configuration - https://github.com/emberjs/data/blob/master/addon/-legacy-private/system/store.js#L520

**Comment by toddjordan** [comment]
Verified, additional calls to that json are loaded from disk cache, reducing the need for this update.
